portlet-security-util/src/main/java/org/jasig/web/filter/SimpleCorsFilter.java

```diff
@@ -0,0 +1,133 @@
+package org.jasig.web.filter;
+
+import java.io.IOException;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * Default behavior for this filter is to allow origins from * (ie anywhere). It is not reccomended to use this default 
+ * behavior except for development testing. When configuring the filter, the filter configuration parameters will 
+ * overwrite any value set in a spring or other configuration file doing dependency injection and initialization.  
+ * 
+ * Config within web.xml:
+ * 
+ * <filter>
+ *   <filter-name>Simple CORS Filter</filter-name>
+ *   <filter-class>org.jasig.web.filter.SimpleCorsFilter</filter-class>
+ *   <init-param>
+ *     <!-- Comma separated domain names -->
+ *     <param-name>allowOrigin</param-name>
+ *     <param-value>someDomain.org, another.com, jasig.org, apereo.org</param-value>
+ *     
+ *     <param-name>maxAge</param-name>
+ *     <param-value>3600</param-value>
+ *     
+ *     <!-- Comma separated methods allowed -->
+ *     <param-name>allowMethod</param-name>
+ *     <param-value>POST, GET</param-value>
+ *     
+ *     <!--Comma separated list of allowed header values -->
+ *     <param-name>allowHeaders</param-name>
+ *     <param-value>Origin, X-Requested-With, Content-Type, Accept</param-value>
+ *   </init-param>    
+ * </filter>
```

Comment:
I added documentation for instantiating this in Spring. By leaving it as a simple servlet filter, users have the ability to use it as such, or use it in a Spring managed way. I also added another @Component based class that extends the filter to allow for more current Spring ways of instantiation